index.html

```diff
@@ -7,6 +7,46 @@
     <title>Document</title>
 </head>
 <body>
-    <h1>basic-html</h1>
+    
 </body>
-</html>
\ No newline at end of file
+</html><h1>HTML task</h1>
```

Comment:
Please, put all of HTML code between `body` tags. The reason is simple - the `body` tag represents the main content of the page, its _body_